sdk/metric/meter_test.go

```diff
@@ -532,6 +545,106 @@ func TestMeterCreatesInstruments(t *testing.T) {
 	}
 }
 
+func TestMeterWithDropView(t *testing.T) {
+	testCases := []struct {
+		name string
+		fn   func(*testing.T, metric.Meter)
+	}{
+		{
+			name: "SyncInt64Count",
+			fn: func(t *testing.T, m metric.Meter) {
+				ctr, err := m.Int64Counter("sint")
+				assert.NoError(t, err)
+				c, ok := ctr.(x.EnabledInstrument)
+				assert.True(t, ok)
+				assert.False(t, c.Enabled())
```

Comment:
this code is common for all test cases. it can be generalized

Reply:
I tried to be more specific here https://github.com/open-telemetry/opentelemetry-go/pull/6016#discussion_r1871047845